applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -14,32 +14,43 @@
 #define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
 
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
-#define LE_AUDIO_PRES_DELAY_US 10000u
+
+#if (CONFIG_AUDIO_SAMPLE_RATE_48000_HZ)
+#define BT_AUDIO_CODEC_CONFIG_FREQ BT_CODEC_CONFIG_LC3_FREQ_48KHZ
+#elif (CONFIG_AUDIO_SAMPLE_RATE_24000_HZ)
+#define BT_AUDIO_CODEC_CONFIG_FREQ BT_CODEC_CONFIG_LC3_FREQ_24KHZ
+#elif (CONFIG_AUDIO_SAMPLE_RATE_16000_HZ)
+#define BT_AUDIO_CODEC_CONFIG_FREQ BT_CODEC_CONFIG_LC3_FREQ_16KHZ
+#endif /* (CONFIG_AUDIO_SAMPLE_RATE_48000_HZ) */
+
+#define BT_AUDIO_LC3_PRESET_CONFIGURABLE(_loc, _stream_context)                                    \
+	BT_AUDIO_LC3_PRESET(                                                                       \
+		BT_CODEC_LC3_CONFIG(BT_AUDIO_CODEC_CONFIG_FREQ, BT_CODEC_CONFIG_LC3_DURATION_10,   \
```

Comment:
Make a BT_AUDIO_(...) config for Frame Duration to keep consistency?

Reply:
Then we open the door for changing the duration to other values before we have changed the rest of the code. We can do it in a later PR